Pokedex_ios/Utilities/Extentions/UIColor.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  UIColor.swift
+//  Pokedex_ios
+//
+//  Created by adri on 7/19/22.
+//
+import UIKit
+import Foundation
+
+extension UIColor {
+    class func applyButtonColor() -> UIColor {
+        return UIColor(named: "apply-button") ?? UIColor.systemGray6
```

Comment:
also use ConstantVariables.applyButtonColor instead of the string

Reply:
Corrected :)